testsuite/tests/typing-modes/stack.ml

```diff
@@ -36,9 +36,9 @@ let f () =
 Line 2, characters 18-26:
 2 |   let g = stack_ ((42, 42) : _ @ global ) in
                       ^^^^^^^^
-Error: This value is "local"
+Error: The allocation is "local"
        because it is "stack_"-allocated.
-       However, the highlighted expression is expected to be "global".
```

Comment:
I think "the highlighted expression is" flows better than just "the highlighted is" here.